application/app/controllers/application_controller.rb

```diff
@@ -1,4 +1,11 @@
 class ApplicationController < ActionController::Base
   # Only allow modern browsers supporting webp images, web push, badges, import maps, CSS nesting, and CSS :has.
   allow_browser versions: :modern
+  before_action :load_user_settings
+
+  private
+
+  def load_user_settings
+    Current.user_settings = UserSettings.new
+  end
```

Comment:
User settings is used to store the active project.
This can be used to store Dataverse data and credentials that the user has configured